data.sql

```diff
@@ -5,4 +5,10 @@ VALUES
 	(1, 'Agumon', 'Feb 3, 2020', 0, TRUE, 10.23),
 	(2, 'Gabumon', 'Nov 15, 2018', 2, TRUE, 8),
 	(3, 'Pikachu', 'Jan 7, 2021', 1, FALSE, 15.04),
-	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11)
+	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11),
+	(5, 'Charmander', 'Feb 8, 2020', 0, FALSE, 11),
+	(6, 'Plantmon', 'Nov 15, 2022', 2, TRUE, 5.7),
+	(7, 'Squirtle', 'Apr 2, 1993', 3, FALSE, 12.13),
+	(8, 'Angemon', 'Jun 12, 2005', 1, TRUE, 45),
```

Comment:
* Kindly correct the highlighted weights as it's a negative one (ie -11, -5.7, -12.13, -45 ) [Check the requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Animal%3A%20His%20name,escape%203%20times.)